python/src/uagents/experimental/chatagent/ai.py

```diff
@@ -0,0 +1,123 @@
+import json
+from typing import Any, Literal
+
+from litellm import completion
+from pydantic import BaseModel, ConfigDict
+from uagents.experimental.chatagent.tools import Tool
+
+OPENAI_API_URL = "https://api.openai.com/v1"
+ANTHROPIC_API_URL = "https://api.anthropic.com/v1/messages"
+ASIONE_API_URL = "https://api.asi1.ai/v1"
+
+DEFAULT_TEMPERATURE = 0.0
+DEFAULT_MAX_TOKENS = 1024
+
+
+class LLMParams(BaseModel):
+    temperature: float = DEFAULT_TEMPERATURE
+    max_tokens: int = DEFAULT_MAX_TOKENS
+    tool_choice: str = "auto"
+    system_prompt: str = (
+        "You are an assistant running inside an on-chain agent. "
+        "If one of the provided tools can fulfill the request, "
+        "call it with correct JSON arguments. "
+        "If none apply, answer in plain text to the user."
+    )
+    model_config = ConfigDict(extra="allow")
+
+
+class LLMConfig(BaseModel):
+    provider: str = Literal["asi1", "openai", "anthropic"]
+    api_key: str
+    model: str
+    url: str
+    parameters: LLMParams
+
+
+openai_config = LLMConfig(
+    provider="openai",
+    api_key="YOUR_OPENAI_API_KEY",
+    model="gpt-4o-mini",
+    url=OPENAI_API_URL,
+    parameters=LLMParams(),
+)
+
+claude_config = LLMConfig(
+    provider="anthropic",
+    api_key="YOUR_ANTHROPIC_API_KEY",
+    model="claude-3-5-haiku-latest",
+    url=ANTHROPIC_API_URL,
+    parameters=LLMParams(),
+)
```

Comment:
Since these are unused configuration settings, I'd suggest moving these to a README.